### PR TITLE
[Ready] Renamings from review

### PR DESCRIPTION
**Pull request by aenario**
Essentially

- puregetters -> getters
- reducers/_store -> redux_store
- reducers/root -> reducers
- some allattached -> snake_case
- components/utils -> ./libs

### CODE REVIEW

**Comment by misstick** [comment]
@aenario : is this ready for merge? could you add a status to you PR such as adding a prefix to its title? (ie. [RFR], [WIP] etc.)

File: client/app/components/layout.coffee [hunk 1]
```diff
@@ -7,7 +7,7 @@ Menu                  = React.createFactory require './menu/menu'
 Modal                 = React.createFactory require './modal'
 ToastContainer        = React.createFactory require './toast_container'
 Tooltips              = React.createFactory require './tooltips-manager'
```
Comment (m4dz) [inline client/app/components/layout.coffee]: So… `tooltips-manager` or `tooltips_manager` x) ? Just kidding, let's take it as it ;)

File: client/app/libs/format_message.coffee [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+            html = "<div class='textOnly'>#{text}</div>"
+
+    # Convert text into markdown
+    if html?.length and not text?.length and not displayHTML
+        text = toMarkdown html
+
+    if text?.length
+        # Tranform URL into links
+        # coffeelint:
+        urls = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=\+\$,\w]+@)?[A-Za-z0-9.-]+|(?:www.|[-;:&=\+\$,\w]+@)[A-Za-z0-9.-]+)((?:\/[\+~%\/.\w-_]*)?\??(?:[-\+=&;%@.\w_]*)#?(?:[\w]*))?)/gim
```
Comment (m4dz) [inline client/app/libs/format_message.coffee]: Is it more readable/maintanable in this form:

```coffee
urls = ///
    (
        (
            ([A-Za-z]{3,9}:(?:\/\/)?)
            (?:[-;:&=\+\$,\w]+@)?
            [A-Za-z0-9.-]+
            |
            (?:www.|[-;:&=\+\$,\w]+@)
            [A-Za-z0-9.-]+
        )
        (
            (?:\/[\+~%\/.\w-_]*)?
            \??
            (?:[-\+=&;%@.\w_]*)
            #?(?:[\w]*)
        )?
    )
///gim
```

File: client/app/libs/make_message.coffee [hunk 3]
```diff
@@ -254,11 +311,11 @@ exports.setMessageAsForward = (options) ->
...
-        inReplyTo
-        dateHuman
-        sender
-        text
-        html
+        # inReplyTo
+        # dateHuman
+        # sender
+        # text
+        # html
```
Comment (m4dz) [inline client/app/libs/make_message.coffee]: Do keeping dead code really valuable here?

File: client/app/libs/format_message.coffee (hunk 2 above)
Comment (aenario) [inline client/app/libs/format_message.coffee]: Agreed

File: client/app/libs/make_message.coffee (hunk 3 above)
Comment (aenario) [inline client/app/libs/make_message.coffee]: Not really, this would be reused when reintroducing compose.

**Pull request merged**